sbin/common-functions.sh

```diff
@@ -28,7 +28,7 @@ determineBuildProperties() {
 
     BUILD_TYPE=normal
     DEFAULT_BUILD_FULL_NAME=${OS_KERNEL_NAME}-${OS_MACHINE_NAME}-${BUILD_TYPE}-${JVM_VARIANT}-release
-    BUILD_FULL_NAME=${BUILD_FULL_NAME:-"$DEFAULT_BUILD_FULL_NAME"}
+    export BUILD_FULL_NAME=${BUILD_FULL_NAME:-"$DEFAULT_BUILD_FULL_NAME"}
```

Comment:
Updating the env variable here I guess this isn't a bad thing - this has gone unnoticed i think hope other types of builds work fine